.storybook/manager-head.html

```diff
@@ -1 +1,7 @@
 <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github-gist.min.css">
+
+
+<script>
+  // HACK: change the page title dynamically
+  document.title = "Photon Components"
```

Comment:
You can't actually just add a title tag here...so i did it like this. couldn't figure out how to swap the favicon, but that would be a nice trick too

Reply:
https://github.com/bwinton/SnoozeTabs/blob/master/src/background.js#L349 might be of help, although it's admittedly an ugly way to do it. 😉